kogito-business-rules-quarkus/README.md

```diff
@@ -61,15 +98,79 @@ To make use of this application it is as simple as putting a sending request to
 ```
```

Comment:
Forgot here about swagger-ui:

line 91
`(Dev Mode Only) https://quarkus.io/guides/openapi-swaggerui#use-swagger-ui-for-development`

Reply:
fixed @radtriste